client/my-sites/email/inbox/mailbox-selection-list/style.scss

```diff
@@ -0,0 +1,196 @@
+@import '@wordpress/base-styles/breakpoints';
+@import '@wordpress/base-styles/mixins';
+
+body {
+	background-color: var( --studio-white );
+}
+
+.mailbox-selection-list {
+
+	.mailbox-selection-list__header {
+		.formatted-header__title {
+			font-size: $font-title-large;
+			line-height: $font-headline-medium;
+
+			@include break-large {
+				font-size: $font-headline-medium;
+			}
+		}
+
+		.formatted-header__subtitle {
+			font-size: $font-body;
+			line-height: $font-title-medium;
+		}
+	}
+
+	.mailbox-selection-list__new-mailbox-upsell-container {
+		background: #f9f9f9;
+		align-items: center;
+		justify-content: space-between;
+		position: relative;
+		margin: 20px auto 10px;
+		padding: 10px 15px;
+		width: 240px;
+
+		@include break-medium {
+			width: 342px;
+			display: flex;
+			margin-top: 60px;
+			padding: 16px 24px;
+		}
+
+		@include break-large {
+			width: 660px;
+			display: flex;
+			margin-top: 80px;
+			padding: 16px 24px;
```

Comment:
It shouldn't be necessary to repeat `display: flex;` and `padding: 16px 24px;` as the breakpoints are for screens wider than that width, so the same values should stay in force.

Reply:
I suspected. Now confirmed - thanks.